guides/source/error_reporting.md

```diff
@@ -8,22 +8,16 @@ This guide introduces ways to manage exceptions that occur in Ruby on Rails appl
 After reading this guide, you will know:
 
 * How to use Rails' error reporter to capture and report errors.
-
-(Because error reporting libraries are mainly for production use, this guide is mostly for production environment too.
-However, you may want to test related setup in development by temporarily activating the libraries locally.)
```

Comment:
Removed this blurb as it's confusing. The Rails error reporter works in every environment, but this made it seem like it only works in production. It's really unnecessary.

Reply:
I asked for this section. Yes you can configure error reporters in all environments, but the use case is mostly in production or staging to report to services such as Sentry &co.

Reply:
Okay. Can I move it to the point where we mention error services, though? I think it's more relevant there. I found it really confusing up here and had to double check whether the Rails error reporter cared about my environment, even though I had followed the implementation. 😅

Reply:
Added:

> Note: The Rails error-reporter will always call registered subscribers, regardless of your environment. However, many error-reporting services only report errors in production by default. You should configure and test your setup across environments as needed.